instagram/src/App.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import './App.css';
+import PostsPage from './components/PostContainer/PostsPage';
+import Authenticate from './components/Authenticate/Authenticate';
+
+class App extends Component {
+  constructor() {
```

Comment:
Should be able to get rid of this since the constructor for a React component is called before it is mounted. Since this is not called before it is mounted, there is no need for it.

Reply:
Correct! It works still!